app/components/question.js

```diff
@@ -80,23 +82,37 @@ class Question extends React.PureComponent {
       difficulty: question.difficulty
     });
   }
-  
-  cancelChanges() {  
+
+  cancelChanges() {
     this.rollbackState(this.props.obj);
     this.props.addOrRemoveQuestionAnswer(this.state.answers, this.props.id);
-    this.props.changeSelectedAnswer(this.props.id, this.state.correctAnswer);    
+    this.props.changeSelectedAnswer(this.props.id, this.state.correctAnswer);
     this.props.closePanel();
   }
 
   saveChanges() {
     this.setState({ answers: this.props.obj.answers });
     this.setState({ correctAnswer: this.props.obj.correctAnswer });
-    this.props.changeQuestionName(this.state.text, this.props.id);   
-    this.props.changeHintQuestion(this.state.hint, this.props.id);    
-    this.props.changeQuestionDifficulty(this.state.difficulty, this.props.id);    
+    this.props.changeQuestionName(this.state.text, this.props.id);
+    this.props.changeHintQuestion(this.state.hint, this.props.id);
+    this.props.changeQuestionDifficulty(this.state.difficulty, this.props.id);
     this.props.closePanel();
   }
 
+  validateQuestion() {
+    if (this.state.text == '') {
+      this.setState({
+        validText: 'error',
+        textMessage: 'This field can\'t be empty'
```

Comment:
en este caso como tiene una comilla simple en el medio del string, ponele comillas dobles, onda "This field can\'t be empty"